AUTHORS

```diff
@@ -263,3 +263,4 @@ Dan Powell <dan@abakas.com>
 Mariana Araújo <simbelm.ne@gmail.com>
 Muhammad Ayub Khan <ayub.khan@arbisoft.com>
 Kaloian Doganov <doganov@gmail.com>
+Sanford Student <sstudent@edx.org>
```

Comment:
Make sure your editor is configured to put a newline at the end of files.  It seems to be leaving them off right now.